.vscode/launch.json

```diff
@@ -40,6 +40,33 @@
 				"${workspaceFolder}/../klighd-vscode/applications/klighd-vscode/dist/**/*.js"
             ],
         },
+        {
+            "name": "Launch VS Code Extension (PASTA)",
```

Comment:
To conform with the naming schemes of our other launch configurations, this should probably include "(Socket)" and indicate that not only the PASTA extension is launched, but also the KIELER extension.